apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -174,11 +176,22 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
     }
   }
 
-  private async _initializePluginsAsync(pluginSpecifiers: ReadonlyArray<string>): Promise<void> {
+  private _getPluginArgumentValues(args: string[] = process.argv): string[] {
+    // This is a rough parsing of the --plugin parameters
```

Comment:
@iclanton  I still find it weird that plugins are specified via the command line, rather than a config file.  Typically a build script is not going to produce correct results unless the correct set of plugins is loaded.

Reply:
I could see us removing this eventually. We're using this internally while transitioning off gulp, though.